src/modules/call/call.service.ts

```diff
@@ -48,8 +50,8 @@ export class CallService {
     return meetFeature;
   }
 
-  async createCallToken(userUuid: string) {
-    const meetFeatures = await this.getMeetFeatureConfigForUser(userUuid);
+  async createCallToken(user: User | UserTokenData['payload']) {
```

Comment:
Just curious, what is the difference between `User` and `UserTokenData['payload]`? Is it the same? Or `User` is for authenticated users and `UserTokenData['payload']` is for anonymous users?